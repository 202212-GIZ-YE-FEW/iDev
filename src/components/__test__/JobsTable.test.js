import renderer from "react-test-renderer";

import JobsTable from "../JobsTable";

it("renders correctly", () => {
    const tree = renderer.create(<JobsTable />).toJSON();
    expect(tree).toMatchSnapshot();
});
