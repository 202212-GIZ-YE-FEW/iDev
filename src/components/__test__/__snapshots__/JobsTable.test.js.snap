// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className=" p-4 flex flex-row cursor-pointer shadow-sm "
>
  <div
    className="basis-3/4 flex flex-col"
  >
    <h3
      className="lg:text-2xl text-lg font-medium  self-start text-[#06b6d4]"
    />
    <p
      className="self-start lg:text-base text-sm text-[#b2a7a7] "
    />
  </div>
  <div
    className="basis-1/4  flex flex-col items-stretch ps-4"
  >
    <h3
      className="lg:text-xl text-base leading-8 self-end text-[#06b6d4]"
    />
    <p
      className=" self-end lg:text-base text-sm text-[#b2a7a7]"
    />
  </div>
</div>
`;
